OpenRA.Game/FieldLoader.cs

```diff
@@ -321,6 +322,24 @@ public static object GetValue(string fieldName, Type fieldType, string value, Me
 					return InvalidValueAction(value, fieldType, fieldName);
 				}
 			}
+			else if (fieldType == typeof(ImageFormat))
+			{
+				switch (value.ToLowerInvariant())
+				{
+					case "bmp":
+						return ImageFormat.Bmp;
+					case "gif":
+						return ImageFormat.Gif;
+					case "jpeg":
+					case "jpg":
+						return ImageFormat.Jpeg;
+					case "tiff":
+					case "tif":
```

Comment:
Seems redundant to me.  Nobody is ever going to want anything except png and maybe jpg.  Even if somebody did, for some obscure purpose, they could losslessly convert the png.